packages/menlo/manifest.yml

```
@@ -0,0 +1,81 @@
+format_version: 2.5.1
+name: menlo
+title: "Menlo Security"
+version: 0.1.0
+source:
+  license: "Elastic-2.0"
+description: "Collect logs from Menlo Security products with Elastic Agent"
+type: integration
+categories:
+  - monitoring
+  - network
+  - security
+conditions:
+  kibana:
+    version: "^8.11.0"
+  elastic:
+    subscription: "basic"
```

Remember to add the `capabilities` field if this package requires it for Serverless.
For instance, other security packages define this:
https://github.com/elastic/integrations/blob/0da0ea5c0edff9412757495b5afafa7778fb7288/packages/ded/manifest.yml#L18

```suggestion
  elastic:
    subscription: "basic"
    capabilities:
      - security
```

This would make this package available in Serverless just for Security projects (and not for Observability projects). cc @elastic/security-service-integrations